apps/android/app/src/main/java/chat/simplex/app/views/chat/ChatView.kt

```diff
@@ -375,8 +379,8 @@ fun BoxWithConstraintsScope.ChatItemsList(
 
   Spacer(Modifier.size(8.dp))
 
+  val reversedChatItems by remember { derivedStateOf { chatItems.reversed() } }
```

Comment:
This will be a very slow operation on a long chat - I've seen it even on a much faster iPhone.

If the reversed list is really needed it has to be maintained, and not created on demand from the whole list.

Reply:
seems fast enough on Android